### PR TITLE
Connection option

### PR DESCRIPTION
**Pull request by pyatnitsev**
[+] new Parameter `connection` to the DoctrineMigrationsBundle that accept connection name like --db option.

### CODE REVIEW

**Comment by pyatnitsev** [comment]
it's related to #263

**Comment by pyatnitsev** [comment]
@jwage, I finish it. Maybe some tests needed, please review.

**Comment by DanilPrussov** [comment]
it's really useful. waiting it)

File: Tests/Command/DoctrineCommandTest.php [hunk 1]
```diff
@@ -88,6 +88,7 @@ private function getContainer() : ContainerBuilder
             'doctrine_migrations.organize_migrations' => Configuration::VERSIONS_ORGANIZATION_BY_YEAR,
             'doctrine_migrations.custom_template' => null,
             'doctrine_migrations.all_or_nothing' => false,
+            'doctrine_migrations.connection' => 'connection',
```
Comment (SenseException) [inline Tests/Command/DoctrineCommandTest.php]: Your test should use the new configuration `connection` instead of this parameter.

**Comment by PReimers** [comment]
@pyatnitsev any news regarding the issue explained in the code review?

**Comment by pyatnitsev** [comment]
@PReimers, could you help me with test for my PR? I can't write this test :(

**Comment by PReimers** [comment]
@pyatnitsev 
I'll take look into the missing test later this week 😉

**Comment by pyatnitsev** [comment]
@PReimers, as I can see, I removed source-repository of this PR. Should I create a new one with this changes to continue work with it? (And reopen new PR maybe? )

**Comment by goetas** [comment]
implemented in https://github.com/doctrine/DoctrineMigrationsBundle/pull/278

**Pull request closed without merging**